client/app/components/ActivityLogComp/index.js

```diff
@@ -44,7 +53,8 @@ class ActivityLogComp extends Component{
                 Type: this.state.type,
```

Comment:
No need to take extra eddittedLog variable, instead you can pass now Activity object of state.